src/coprocessor/dag/rpn_expr/types/expr_builder.rs

```diff
@@ -0,0 +1,698 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![allow(dead_code)]
```

Comment:
Can we remove it? Or better to add a TODO here?

Reply:
Currently the expression built is not used, so it is a dead code. We will remove it when the evaluation part get merged.